core/src/banking_stage.rs

```diff
@@ -93,7 +89,7 @@ pub struct BankingStageStats {
     new_tx_count: AtomicUsize,
     dropped_packet_batches_count: AtomicUsize,
     dropped_packets_count: AtomicUsize,
-    dropped_duplicated_packets_count: AtomicUsize,
+    pub(crate) dropped_duplicated_packets_count: AtomicUsize,
```

Comment:
per other nitpicking, instead of making these two stats `pub(crate)`, might worth consider to remove them from `BankingStageStats`.